maths/sphenic_number.dart

```diff
@@ -0,0 +1,50 @@
+var arr = new List<int>.filled(1001, 1, growable: false);
```

Comment:
You are holding just `0`/`1` in this List? In this case it would be clearer to use `List<bool>`

Reply:
yes sir